master/getting-started/rkt/installation/manual.md

```diff
@@ -0,0 +1,173 @@
+---
+title:  Manual installation of Calico with rkt
+---
+
+This tutorial describes how to manually configure a working environment for
+a cluster of Calico-rkt enabled nodes.
+
+To run through the tutorials, you will require at least two nodes, which we have
```

Comment:
It is still true (I'm talking about the tutorials) and I deliberately put it there, but it feels extremely badly worded and out of place - so I'll remove :-)